doorway-ui-components/src/page_components/listing/ListingCard.scss

```diff
@@ -15,27 +15,39 @@
   margin: var(--margin);
   position: relative;
 
-  @media (min-width: $screen-lg) {
+  @media (min-width: $screen-xl) {
     --max-width: var(--max-width-large-screen);
   }
 
   &:first-of-type {
     margin-top: var(--bloom-s12);
   }
 }
+.listings-row_figure {
+  display: flex;
+
+  .image_card {
+    flex: 1;
+  }
+
+  .image-card__inner,
+  img {
+    height: 100%;
+  }
+  flex: 1;
```

Comment:
Is this syntax right? Should it be 

.image-card__inner {
  img {
    height: 100%;
  }
  flex: 1;
}

Reply:
Synced offline, this is intended so it styles both elements.